File: tests/twiggs_money_flow_test.py
import numpy as np
from taflow import TwiggsMoneyFlow
def test_twiggs_money_flow_lifecycle():
    s=TwiggsMoneyFlow(np.array([],float),np.array([],float),np.array([],float),np.array([],float),2);s.extend([2,2],[1,1],[1.5,1.5],[10,10]);assert s.value is not None;s.reset();assert len(s)==0

File: crates/taflow-core/src/indicators/twiggs_money_flow_test.rs
use super::twiggs_money_flow::TwiggsMoneyFlow;
#[test]
fn lifecycle() {
    let mut s = TwiggsMoneyFlow::new(2).unwrap();
    s.append(2.0, 1.0, 1.5, 10.0);
    assert!(s.append(2.0, 1.0, 1.5, 10.0).is_some());
    s.reset();
    assert!(s.value().is_none());
}
